source/stylesheets/refills/_laptop.scss

```diff
@@ -0,0 +1,73 @@
+.laptop {
+  $lid-background: darken(#fff, 5);
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.